cmake/Finddpdk.cmake

```diff
@@ -175,7 +184,7 @@ if (dpdk_FOUND AND NOT (TARGET dpdk))
     set_target_properties (DPDK::dpdk
       PROPERTIES
         INTERFACE_INCLUDE_DIRECTORIES "${dpdk_PC_INCLUDE_DIRS}"
-        INTERFACE_LINK_LIBRARIES "${_dpdk_libraries}"
+        INTERFACE_LINK_LIBRARIES "${_dpdk_libraries} ${dpdk_dependencies}"
```

Comment:
i think the argument for `INTERFACE_LINK_LIBRARIES` should be a list instead of space-separated values. see https://cmake.org/cmake/help/latest/prop_tgt/INTERFACE_LINK_LIBRARIES.html and in CMake, a list is semicolon-separated. see https://cmake.org/cmake/help/latest/command/list.html, so probably, we should put something like:
```cmake
INTERFACE_LINK_LIBRARIES "${_dpdk_libraries};${dpdk_dependencies}"
```
?

Reply:
Thanks for noticing this, I wonder how it worked on my side. Fixed now.